src/main/kotlin/no/nav/familie/ef/sak/beregning/OmregningService.kt

```diff
@@ -50,38 +46,37 @@ class OmregningService(private val behandlingService: BehandlingService,
     private val logger = LoggerFactory.getLogger(javaClass)
 
     @Transactional(propagation = Propagation.REQUIRES_NEW)
-    fun utførGOmregning(ferdigstiltBehandlingIdMedGammelG: UUID) {
+    fun utførGOmregning(fagsakId: UUID) {
 
         feilHvisIkke(featureToggleService.isEnabled("familie.ef.sak.omberegning")) {
             "Feature toggle for omberegning er disabled"
         }
 
-        val fagsak = fagsakRepository.finnFagsakTilBehandling(ferdigstiltBehandlingIdMedGammelG)
 
-        val fagsakMedOppdatertPersonIdent = fagsakService.fagsakMedOppdatertPersonIdent(fagsak.id)
+        val fagsakMedOppdatertPersonIdent = fagsakService.fagsakMedOppdatertPersonIdent(fagsakId)
         val sisteBehandling =
                 behandlingRepository.finnSisteBehandlingSomIkkeErBlankett(fagsakMedOppdatertPersonIdent.stønadstype,
                                                                           fagsakMedOppdatertPersonIdent.personIdenter.map {
                                                                               it.ident
                                                                           }.toSet())
 
         feilHvis(sisteBehandling == null) {
-            "Kan ikke omberegne: Fant ikke sisteBehandlingId for fagsak ${fagsak.id}"
+            "Kan ikke omberegne: Fant ikke sisteBehandlingId for fagsak $fagsakId"
         }
         feilHvisIkke(sisteBehandling.erAvsluttet()) {
             "Kan ikke omberegne fagsak med åpen behandling, id=${sisteBehandling.id}"
         }
 
         val behandling = behandlingService.opprettBehandling(behandlingType = BehandlingType.REVURDERING,
-                                                             fagsakId = fagsak.id,
+                                                             fagsakId = fagsakId,
                                                              behandlingsårsak = BehandlingÅrsak.G_OMREGNING)
-        logger.info("G-omregner fagsak=${fagsak.id} behandling=${behandling.id} ")
+        logger.info("G-omregner fagsak=$fagsakId behandling=${behandling.id} ")
 
 
-        val forrigeTilkjentYtelse = ytelseService.hentForBehandling(ferdigstiltBehandlingIdMedGammelG)
+        val forrigeTilkjentYtelse = ytelseService.hentForBehandling(sisteBehandling.id)
```

Comment:
Hvis siste behandling er "AVSLAG" - da har den kanskje ikke en tilkjent ytelse? 

Vi trenger vel to behandlinger her:
- Siste iverksatte behandling (for å hente ut tilkjent ytelse)
- Siste behandling (for å sjekke at den er avsluttet) 


Kanskje vi ikke trenger å "hente ut" siste behandling som sådan, men validere at siste behandling er avsluttet?